tools/README.md

```diff
@@ -0,0 +1,35 @@
+# How to Build
+
+## Requirements
+
+Environment Variables:
+- `AWS_ACCESS_KEY`
+- `AWS_SECRET_KEY`
+- `GITHUB_API_TOKEN`
+
+## LogDNA Agent for Debian Systems
+LogDNA Agent can be built and released for Debian systems using `debian.sh` script which has the following steps:
+1. Install Dependencies:
+	* Install `nexe` which is for compiling the files into the executable;
+	* Install `fpm` for packaging it up and `deb-s3` for pushing the packages into the specified S3 bucket;
+	* Install `ghr` for creating and updating tag-based GitHub Releases.
+2. Compile the source code into the executable;
+3. Package the executable into the Debian package;
+4. Create / Update the GitHub Release for the specified tag by uploading the Debian package;
+5. Publish the Debian package into the specified S3 bucket.
+
+## LogDNA Agent for RedHat Systems
+LogDNA Agent can be built and released for RedHat systems using `redhat.sh` script which has the following steps:
+1. Install Dependencies:
+	* Install `nexe` which is for compiling the files into the executable;
+	* Install `fpm` for packaging it up
+	* Clone `rpm-s3` repo for pushing the packages into the specified S3 bucket;
+	* Install `ghr` for creating and updating tag-based GitHub Releases.
+2. Compile the source code into the executable;
+3. Package the executable into the RedHat package;
+4. Create / Update the GitHub Release for the specified tag by uploading the RedHat package;
```

Comment:
Ditto.  I don't think Release is a proper name.

Reply:
@darinspivey, updated but in this step, it's all about creating `GitHub Draft Release`